inspire_dojson/hep/rules/bd5xx.py

```diff
@@ -221,15 +231,11 @@ def _get_material(value):
 
         return MATERIAL_MAP.get(value.get('3'))
 
-    license_value = force_list(value.get('a'))
-    # We strip away redundant 'Open Access' string
-    license_value = [val for val in license_value if license_value != 'Open Access']
```

Comment:
Note the subtle bug: because `license_value` has been forced to a `list` in the line above then it can't ever be equal to the string `'Open Access'`.